Apps/Admin/Server/Models/CovidSupport/CovidAssessmentRequest.cs

```diff
@@ -133,6 +133,12 @@ public CovidAssessmentRequest()
     [JsonPropertyName("hasChronicConditionDiagnoses")]
     public CovidTherapyAssessmentOption HasChronicConditionDiagnoses { get; set; }
 
+    /// <summary>
+    /// Gets or sets the has consented to share information option.
+    /// </summary>
+    [JsonPropertyName("consentToSendCC")]
+    public CovidTherapyAssessmentOption consentToSendCC { get; set; }
```

Comment:
First letter needs to be capital. "ConsentToSendCC "

Reply:
updated, thank you!